modules/simple_forms_api/app/form_mappings/vba_20_10206.json.erb

```diff
@@ -1,33 +1,33 @@
 {
-  "form1[0].#subform[7].First_Name[0]": "<%= data.dig('full_name', 'first') %>",
-  "form1[0].#subform[7].Middle_Initial1[0]": "<%= data.dig('full_name', 'middle') %>",
-  "form1[0].#subform[7].Last_Name[0]": "<%= data.dig('full_name', 'last') %>",
-  "form1[0].#subform[7].FirstThreeNumbers[0]": "<%= data.dig('citizen_id', 'ssn')&.[](0..2) %>",
-  "form1[0].#subform[7].SecondTwoNumbers[0]": "<%= data.dig('citizen_id', 'ssn')&.[](3..4) %>",
-  "form1[0].#subform[7].LastFourNumbers[0]": "<%= data.dig('citizen_id', 'ssn')&.[](5..8) %>",
-  "form1[0].#subform[7].Alien_Registration_Number_If_applicable[0]": "<%= data.dig('non_citizen_id', 'arn') %>",
-  "form1[0].#subform[7].VA_File_Number_If_Applicable[0]": "<%= data['citizen_id'] ? data.dig('citizen_id', 'va_file_number') : data.dig('non_citizen_id', 'va_file_number') %>",
-  "form1[0].#subform[7].Place_Of_Birth[0]": "<%= data['place_of_birth'] %>",
-  "form1[0].#subform[7].CurrentMailingAddress_City[0]": "<%= data.dig('address', 'city') %>",
-  "form1[0].#subform[7].CurrentMailingAddress_NumberAndStreet[0]": "<%= data.dig('address', 'street') %>",
-  "form1[0].#subform[7].CurrentMailingAddress_ApartmentOrUnitNumber[0]": "<%= data.dig('address', 'street2') %>",
-  "form1[0].#subform[7].CurrentMailingAddress_StateOrProvince[0]": "<%= data.dig('address', 'state') %>",
-  "form1[0].#subform[7].CurrentMailingAddress_Country[0]": "<%= data.dig('address', 'country') %>",
-  "form1[0].#subform[7].CurrentMailingAddress_ZIPOrPostalCode_FirstFiveNumbers[0]": "<%= data.dig('address', 'postal_code')&.[](0..4) %>",
-  "form1[0].#subform[7].CurrentMailingAddress_ZIPOrPostalCode_LastFourNumbers[0]": "<%= data.dig('address', 'postal_code')&.[](5..8) %>",
-  "form1[0].#subform[7].Year[0]": "<%= data['date_of_birth']&.[](0..3) %>",
-  "form1[0].#subform[7].Month[0]": "<%= data['date_of_birth']&.[](5..6) %>",
-  "form1[0].#subform[7].Day[0]": "<%= data['date_of_birth']&.[](8..9) %>",
-  "form1[0].#subform[7].AreaCode[0]": "<%= data['home_phone']&.gsub('-', '')&.[](0..2) %>",
-  "form1[0].#subform[7].FirstThreeNumbers[1]": "<%= data['home_phone']&.gsub('-', '')&.[](3..5) %>",
-  "form1[0].#subform[7].LastFourNumbers[1]": "<%= data['home_phone']&.gsub('-', '')&.[](6..9) %>",
+  "form1[0].#subform[7].First_Name[0]": "<%= form.data.dig('full_name', 'first') %>",
```

Comment:
This kind of change comprises the bulk of the changes in this PR and I don't know how to handle the danger error that results. We need to make this switch over hundreds of lines at once or else things won't work.